test/user.test.js

```diff
@@ -401,6 +401,29 @@ describe('User', function() {
     var pass73Char = pass72Char + '3';
     var passTooLong = pass72Char + 'WXYZ1234';
 
+    it('rejects empty passwords creation', function(done) {
+      try {
+        User.create({email: 'b@c.com', password: ''}, function(err) {
+          if (err) return done(err);
+          done(new Error('User.create() should have thrown an error.'));
+        });
+      } catch (e) {
+        expect(e).to.match(/Invalid password/);
+        done();
+      }
+    });
+
+    it('rejects updating with empty password', function(done) {
+        User.create({email: 'blank@c.com', password: pass72Char}, function(err, userCreated) {
+          if (err) return done(err);
+          userCreated.updateAttribute('password', '', function(err, userUpdated) {
+            assert(err);
+            expect(err).to.match(/Invalid Password/);
```

Comment:
Please verify `err.code`, not the message.

I think both lines 420 and 421 can be compressed to the following:

```js
expect(err).to.have.property('code', 'INVALID_PASSWORD');
```